lib/internals/handler.ts

```diff
@@ -70,7 +87,8 @@ export function Handler(method?: HttpVerb): MethodDecorator {
         if (err instanceof HttpException) {
           res.status(err.statusCode ?? 500).json({
             statusCode: err.statusCode ?? 500,
-            message: err.message,
+            error: err.message ?? 'An unknown error occurred.',
+            errors: err.errors,
             stack: 'stack' in err && process.env.NODE_ENV === 'development' ? err.stack : undefined
           });
         } else {
```

Comment:
Think it would be good to return error in the `else` statement in the same format as above, just with the defaults (`'An unknown error occurred.'`) if something unexpected happened and threw another kind of error. In this scenario `stack` can also be handy for debugging.